packages/salesforce/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.14.2"
```

```suggestion
- version: "0.15.0"
```
As this is a major enhancement.